Marlin/src/HAL/HC32/sdio.cpp

```diff
@@ -54,7 +54,7 @@
     fn                                              \
   }
 
-stc_sd_handle_t *handle;
+stc_sd_handle_t *handle = nullptr;
```

Comment:
I would prefer to see the SDIO changes in a separate PR. That way if there are problems with either the SDIO or SERIAL_DMA features they can be readily reverted independent of one another.

I'm not really going to hold the line on these, the changes aren't all that complicated and Marlin has a long history of large commits of unrelated changes.

Reply:
ok, i'll keep unrelated changes in separate PRs going forward.